pkg/config/predicates.go

```diff
@@ -25,8 +25,13 @@ import (
 func Predicates() predicate.Predicate {
 	return predicate.Funcs{
 		UpdateFunc: func(evt event.UpdateEvent) bool {
-			if config, ok := evt.ObjectNew.(*dw.DevWorkspaceOperatorConfig); ok {
-				syncConfigFrom(config)
+			if newConfig, ok := evt.ObjectNew.(*dw.DevWorkspaceOperatorConfig); ok {
+				syncConfigFrom(newConfig)
+				if oldConfig, ok := evt.ObjectOld.(*dw.DevWorkspaceOperatorConfig); ok {
```

Comment:
Why do we need this?

Reply:
To trigger certificate injection on the `TLSCertificateConfigmapRef` config change